recipes/atomgpt/LICENSE

```diff
@@ -0,0 +1,28 @@
+# Terms of Use
```

Comment:
Please remove this one and use the one in the tarball.

Reply:
Resolved

Reply:
You are still adding the file. The file is in the tarball, no need to add it twice.